src/doc/src/reference/unstable.md

```diff
@@ -1407,7 +1407,7 @@ fn main() {}
 
 A user may optionally specify a manifest in a `cargo` code fence in a module-level comment, like:
 ```rust
-#!/usr/bin/env cargo
+#!/usr/bin/env cargo +nightly -Zscript
```

Comment:
This is platform dependent.  On mac, this will work.  On Linux, this will require `-S`.

A simple approach is we recommend people creating a `nargo`
```sh
#!/usr/bin/env sh

cargo +nighly -Zscript $@
```

and then update the example to
````rust
```rust
#!/usr/bin/env nargo
````

Reply:
do you mean specifically when used in the shebang? looks like `+nightly` and `-Z` is used quite a bit in the other examples in this file :)

Reply:
@yerke since this doesn't need to be held to the same quality of documentation as production documentation, could you add `-S` and then we can merge this.

Reply:
Added `-S`. Made sure that it works on Mac.